ClientApp/src/app/app.module.ts

```diff
@@ -21,21 +29,33 @@ import { AboutComponent } from './components/about/about.component';
       CounterComponent,
       FetchDataComponent,
       AboutComponent,
+      AuthComponent
    ],
    imports: [
       BrowserModule.withServerTransition({ appId: 'ng-cli-universal' }),
+      AppRoutingModule,
       HttpClientModule,
+      MaterialModule,
```

Comment:
`MaterialModule` was already imported at [line 56](https://github.com/Brixel/Singer/pull/4/files#diff-406bed22d2c0910f6373fb9869090d88R56). Merge issue?

Reply:
Yes, this is common practice. The performance impact is ok imho.